4.LastFM/css/index.css

```diff
@@ -0,0 +1,139 @@
+html, body {
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
+header {
+  background-color: #1464F6;
+  padding: 12px 0;
+  color: white;
+  font-size: 18px;
+  margin-bottom: 20px;
+  display: flex;
+}
+
+.wrapper {
+  max-width: 1200px;
+  min-width: 1200px;
+  /*width: 100%;*/
+  height: 100%;
+  margin: 0 auto;
+  padding: 0 12px;
+  box-sizing: border-box;
+  display: inherit;
+}
+
+.center {
+  margin: auto 0;
+}
+
+.content {
+  min-height: 100%;
+  padding-bottom: 80px;
+  box-sizing: border-box;
+}
+
+footer {
+  background-color: #1464F6;
+  color: white;
+  font-size: 12px;
+  padding: 8px 0;
+  margin-top: -30px;
+  display: flex;
+}
+
+.search-box {
+  width: auto;
+  height: 28px;
+  display: table;
+  margin: 0 auto;
+  border: 2px solid #1464F6;
+  box-sizing: content-box;
+  border-radius: 5px; 
+}
+
+.search-box input {
+  color: #1464F6;
+  height: 100%;
+  padding-left: 12px;
+  border: 0;
+  width: 300px;
+}
+
+.search-box input:focus {
+  outline: 0;
+}
+
+.search-box input::-webkit-input-placeholder {
+  color: #1464F6;
+}
+
+.search-box button {
+  height: 100%;
+  width: 100px;
+  color: white;
+  background-color: #1464F6;
+  border: 0;
+  cursor: pointer;
+}
+
+.search-box button:focus {
+  outline: 0;
+}
+
+.units-box {
+  padding: 0;
+  margin: 12px -12px 0;
+  display: table;
+  box-sizing: content-box;
+}
+
+.unit {
+  width: 216px;
+  height: 216px;
+  margin: 12px;
+  padding: 9px 20px;
+  border: 1px solid #e2d6fc;
+  box-sizing: border-box;
+  float: left;
+  cursor: pointer;
+}
+
+.unit img {
+  height: 174px;
+  width: 174px;
+  background-color: #d3d9e0;
+}
+
+.unit .title {
+  width: 176px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.paginator {
+  width: 1200px;
+  padding: 9px;
+  font-size: 22px;
+  border: 1px solid black;
+  box-sizing: border-box;
+}
+
+.paginator span {
+  margin: 9px;
+  cursor: pointer;
+}
+
+.current-page {
```

Comment:
I suppose `.current-page` is related to `.paginator` + it's not global element, so selector should be `.paginator .current-page`